dashboard/src/App.vue

```diff
@@ -1,12 +1,18 @@
 <template>
-  <div id="dashboard">
-    <SidebarMenu class="should-be-sidebar" @toggle="toggleSidebar" />
-    <router-view id="routerview" :class="{'sidebar__active': !sidebarClosed }" />
+  <div>
+    <div id="dashboard" v-if="online">
+      <SidebarMenu class="should-be-sidebar" @toggle="toggleSidebar" />
+      <router-view id="routerview" :class="{'sidebar__active': !sidebarClosed }" />
+    </div>
+    <div v-else>
+      <Offline />
```

Comment:
I think we want to keep sidebar, a user could be confused what or where page is at, just saying it's offline

Reply:
Make sense. I'll update this now.